wheatley/rhythm/regression.py

```diff
@@ -214,6 +217,22 @@ def log_warning(message: str) -> None:
                 log_warning(f"{new_peal_speed} is not positive")
             else:
                 self._peal_speed = new_peal_speed
+                # Don't try to calculate the regression line if _blow_interval is 0 (because the lin
```

Comment:
I hope a blow interval of 0 is beacuse we haven't started ringing yet 😃

Reply:
Yep that is indeed the case - the user could change the peal speed whilst Wheatley isn't ringing, and in that case the line will be overwritten so it doesn't matter what we do to it.